### ISSUE TITLE
Artifact ID not AMI ID

### ISSUE THREAD

**Issue by cliffano** [bhourigan/packer-post-processor-ami-registry#1]
I'm getting region:ami_id, instead of just ami_id as the result of artifact.Id() at  https://github.com/bhourigan/packer-post-processor-ami-registry/blob/master/post-processor.go#L36 .

Is the intention to just have the AMI ID from artifact.Id() ?

Edit: fix format

**Comment by bhourigan** [comment]
I'm working on some additional code which will post the resulting AMI IDs to an API, but that's still under development. I don't have a functional version (yet).  It just printing a comma separated list of AMI IDs for now, mainly so I could track down where in the packer data structure this data is stored.

Here's what it looks like for me under packer 0.7.5:

```
amazon-instance-us-west-2-ubuntu (ami-registry): Builder:amazoninstance AMI:eu-west-1:ami-8574faf2,us-east-1:ami-e6a4e68e,us-west-2:ami-9db7ecad
```

**Comment by cliffano** [comment]
Thanks for the confirmation.
Initially wasn't sure if the expectation is to have region:ami_id or just ami_id without region.

**Comment by bhourigan** [comment]
No problem. I am curious - what is your use case? Do you use anything to manage your AMIs or is that an unsolved problem?

**Comment by cliffano** [comment]
I don't have any particular use case actually, I started using Packer a couple of weeks ago and have been reading the source code of Packer and some of its plugins that I encountered.
I was looking for approaches to limit which builders will be supported for a particular post processor plugin, so that's where I saw your 'builtins var' approach in your plugin.

Re AMI management. what are the problems you're trying to solve? I've been attaching tags to the AMIs, so it's easy to look up / group / delete later on.
